pkgs/by-name/gr/graphicsmagick/imagemagick-compatibility.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenvNoCC
+, graphicsmagick
+, makeWrapper
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "graphicsmagick-imagemagick-compat";
+  inherit (graphicsmagick) version;
+
+  outputs = [ "out" "man" ];
+
+  nativeBuildInputs = [ makeWrapper ];
```

Comment:
I think we should use `makeBinaryWrapper` here since it's available on Darwin.

Reply:
Isn't `makeBinaryWrapper` only an issue for GUI applications? This is supposed to be only CLI.

Reply:
Oh, ok, I remember that in macOS non-binary wrappers can't be used in shebangs.

Reply:
Let's get back to the old links...